core/src/main/java/com/alibaba/fescar/core/rpc/netty/AbstractRpcRemotingClient.java

```diff
@@ -321,6 +333,27 @@ public void dispatch(long msgId, ChannelHandlerContext ctx, Object msg) {
         }
     }
 
+    protected void reconnect(String transactionServiceGroup) {
+        List<String> availList = null;
+        try {
+            availList = getAvailServerList(transactionServiceGroup);
+        } catch (Exception exx) {
+            LOGGER.error(exx.getMessage());
```

Comment:
`Failed to get xxxx: {}`

Reply:
Sorry I'm a little confused, but I changed my forked branch code and committed again.

Reply:
Resolved.